py-polars/src/lazy/dsl.rs

```diff
@@ -187,7 +187,7 @@ impl PyExpr {
     pub fn last(&self) -> PyExpr {
         self.clone().inner.last().into()
     }
-    pub fn list(&self) -> PyExpr {
+    pub fn implode(&self) -> PyExpr {
         self.clone().inner.list().into()
```

Comment:
I think we must follow the `list` trace all the way down. It should be called `implode` on the rust side as well now.

Reply:
Will do 👍